### DISCUSSION TITLE
Creating trigger volumes?

### DISCUSSION THREAD

**Discussion by ashelleyPurdue** [blackears/cyclopsLevelBuilder#165] [Q&A] [answered]
Is there a way to create trigger volumes with this plugin, or is that a yet-to-be-added feature?

**Comment by blackears** [comment, marked as answer]
I hadn't thought of adding it, since you can just use Godot's collision volumes.  Do you think Cyclops could do it better in some way?

**Reply by ashelleyPurdue** [reply]
Cyclops's blocks are easier to create and align than Godot's built-in objects (that is, after all, the whole point of the plugin :p).
